extensions/amp-ad/0.1/test/test-concurrent-load.js

```diff
@@ -75,13 +76,27 @@ describes.realWin('concurrent-load', {}, env => {
       installTimerService(win);
     });
 
-    it('should throttle ad loading one per second', () => {
+    it('should throttle ad loading one per second', function* () {
       expect(is3pThrottled(win)).to.be.false;
       incrementLoadingAds(win);
       expect(is3pThrottled(win)).to.be.true;
       clock.tick(999);
+      yield macroTask();
```

Comment:
> Right now, the macroTask isn't testing anything. You've frozen time in test's window.

@jridgewell here we want to yield an event loop so that if the timer has triggered the promise resolver, we wait till the resolution finish.

@zhouyx your suggestion is also treating the 2 ticks differently, which I don't like